monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java

```diff
@@ -647,7 +647,12 @@ public ExecutionIdentifier eval(final RepositoryFile file) {
                         ExecutionIdentifier result = file.readFile(new Func<InputStream, ExecutionIdentifier>() {
                             @Override
                             public ExecutionIdentifier eval(InputStream in) {
-                                return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                try {
+                                    return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                } catch (Exception e) {
+                                    logger.error("The file {} could not be read" + e, file.getQualifiedPath());
```

Comment:
BTw, you should avoid concatenating like this in SLF4J, it should work fine to have the exception as the last argument, as in `logger.warn("The file {} could not be read", file.getQualifiedPath(), e);`

http://slf4j.org/faq.html#paramException